strings.txt

```diff
@@ -16244,3 +16246,19 @@
     tags = ios, android
     en = Unknown Place
     ru = Неизвестное место
+
+  [editor_other_info]
+    tags = ios, android
+    en = Other information about place
```

Comment:
Да зачем, если потом всё равно будут хорошие правильные переводы? Двойная-тройная работа получается.

> On Apr 21, 2016, at 4:02 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/2947#discussion_r60574990>:
> 
> > @@ -16244,3 +16246,19 @@
> >      tags = ios, android
> >      en = Unknown Place
> >      ru = Неизвестное место
> > +
> > +  [editor_other_info]
> > +    tags = ios, android
> > +    en = Other information about place
> На русском тоже лучше добавить сразу. У Игоря есть хорошие базовые варианты на инглише и на русском?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2947/files/44b3f477a158544a38212dbfd95100ddbb3c305c#r60574990>